omnigibson/prims/entity_prim.py

```diff
@@ -1229,8 +1229,7 @@ def kinematic_only(self, val):
                 body's pose / velocities. See https://docs.omniverse.nvidia.com/app_create/prod_extensions/ext_physics/rigid-bodies.html?highlight=rigid%20body%20enabled#kinematic-rigid-bodies
                 for more information
         """
-        for link in self.links.values():
-            link.kinematic_only = val
+        self.root_link.kinematic_only = val
```

Comment:
Is this intended?

Reply:
Yeah -- this is to remove all those joint errors you were getting